cob_twist_controller/include/cob_twist_controller/limiters/limiter_base.h

```diff
@@ -43,7 +44,29 @@ class LimiterBase
         /**
          * Pure virtual method to mark as interface method which has to be implemented in inherited classes.
          * The intention is to implement a method which enforces limits to the q_dot_out vector according to
-         * the calculated joint velocities and / or joint positions.
+         * the calculated joint velocities and / or joint positions and or cartesian twists.
+         * @param q_dot_ik The calculated joint velocities vector which has to be checked for limits.
+         * @param q The last known joint positions.
+         * @return Scaled joint velocities vector.
+         */
+
+    protected:
+        const LimiterParams& limiter_params_;
+};
+
+/// Joint Base class for limiters, defining interface methods.
+class LimiterJointBase
+{
+    public:
+        explicit LimiterJointBase(const LimiterParams& limiter_params) : limiter_params_(limiter_params)
+        {}
```

Comment:
LimiterParams are inherited from LimiterBase class, thus the constructor should look like this:
```
explicit LimiterJointBase(const LimiterParams& limiter_params) : 
    LimiterBase(limiter_params)
{}
```